tests/foreman/ui/test_myaccount.py

```diff
@@ -58,7 +57,8 @@ def setUp(self):
         # Creating user for each test to not mess with default user
         user = User(login=gen_alpha(), password='password')
         # copying missing id because create returns only password's hash once
-        # it is not stored in plain text for security reason
+        # it is not stored in plain text for security reason and original
```

Comment:
any way cannot understand what are the security reasons here to not use the more simple approach with a random password stored in the instance variable, is there any restriction for this? 
```python
self.account_user_password = gen_string('alphanumeric')
```